Dockerfile

```diff
@@ -1,28 +1,19 @@
-FROM docker-registry.eyeosbcn.com/eyeos-fedora21-node-base:latest
+FROM docker-registry.eyeosbcn.com/alpine6-node-base
 MAINTAINER eyeos
 
 ENV WHATAMI ldap
 ENV LDAP_LOG_LEVEL 0
 
-COPY files/monitor /tmp/
-COPY files/hdb /tmp/
-COPY files/base.ldif /tmp/
-COPY files/ldap_rootpwd /tmp/
-COPY start.sh /tmp/
-COPY ldap.sh /tmp/
-COPY files/bind_v2.ldif /tmp/
+COPY alpine-*.list /var/service/
+COPY files/base.ldif files/ldap_rootpwd start.sh ldap.sh files/slapd.conf /tmp/
 
-RUN yum install -y openldap-servers openldap-clients hostname unzip
-RUN mkdir -p /var/run/slapd && touch /var/run/slapd/slapd.pid && \
-    npm install -g eyeos-run-server eyeos-tags-to-dns eyeos-service-ready-notify-cli && \
+RUN /scripts-base/buildDependencies.sh --production --install && \
+    npm install -g --verbose eyeos-run-server eyeos-tags-to-dns eyeos-service-ready-notify-cli && \
```

Comment:
`eyeos-run-server` and `eyeos-tags-to-dns` are present in the base installation.